roles/container-engine/containerd/defaults/main.yml

```diff
@@ -2,7 +2,7 @@
 containerd_storage_dir: "/var/lib/containerd"
 containerd_state_dir: "/run/containerd"
 containerd_systemd_dir: "/etc/systemd/system/containerd.service.d"
-containerd_oom_score: 0
+containerd_oom_score: -999
```

Comment:
We already have tested this `-999` on packet_centos7-flannel-addons-ha job.
So it is fine to change it as the containerd's recommendation.